Firmware/LightpackUSB.c

```diff
@@ -98,6 +98,7 @@ bool CALLBACK_HID_Device_CreateHIDReport(USB_ClassInfo_HID_Device_t* const HIDIn
     // Firmware version
     ReportData_u8[INDEX_FW_VER_MAJOR] = VERSION_OF_FIRMWARE_MAJOR;
     ReportData_u8[INDEX_FW_VER_MINOR] = VERSION_OF_FIRMWARE_MINOR;
+    ReportData_u8[INDEX_FW_VER_UNOFFICIAL] = VERSION_OF_FIRMWARE_UNOFFICIAL;
     return true;
```

Comment:
Yes, before this function is called the whole array is set to zero.
Also, the array is 64 bytes long, so there can't be a buffer overrun.